.github/files/required-review.yaml

```
@@ -22,6 +22,7 @@
   paths:
    - 'projects/packages/**'
    - 'projects/plugins/jetpack/**'
+   - '!projects/packages/search/**'
```

I wonder if it's worth a comment pointing out why we have this line here.
```suggestion
   # Exclude packages managed by other teams, which are covered by other blocks below.
   - '!projects/packages/search/**'
```